### PR TITLE
Admin extensions

### PR DESCRIPTION
**Pull request by mjumbewu**
This should close issue #162 (organizational changes through the admin) and #173 (export affordance).

Some things to be aware of:
* The schema changed. There is a more explicit relationship between users and departments. Previously, users had a string field representing any department they were primary or backup contact for. Now that relationship is represented by primary key fields on departments to users.
* Adding relationships between users and departments in the admin requires that both be created first.
* The "export affordance" is really simple; a link.

### CODE REVIEW

**Comment by waltz** [comment]
I'm not super active on this project and I'm not in a place to merge this, but I want to give a huge thumbs up :+1: to active work on this project.

**Comment by richaagarwal** [comment]
@mjumbewu Thanks for this work!  I pulled the branch down locally and noticed that the /view_requests page doesn't load (in addition 8 test failures) -- is this still a WIP?

**Comment by mjumbewu** [comment]
@richaagarwal The /view_requests route loads fine for me. What does it look like to you? 404? 403? Blank screen?

And yeah, I'm still looking into those failing tests.

**Comment by richaagarwal** [comment]
@mjumbewu Woops, had to start with a fresh database, my bad -- /view_requests is all good.

File: public_records_portal/models.py [hunk 1]
```diff
@@ -31,14 +31,17 @@ class User(db.Model):
 	phone = db.Column(db.String())
 	date_created = db.Column(db.DateTime)
 	password = db.Column(db.String(255))
-	department = db.Column(Integer, ForeignKey("department.id"))
-	current_department = relationship("Department", foreign_keys = [department], uselist = False)
+	department_id = db.Column(Integer, ForeignKey("department.id", use_alter=True, name="fk_department"))
```
Comment (richaagarwal) [inline public_records_portal/models.py]: Is it necessary to do this renaming? If so, a migration would be helpful.

**Pull request merged**